approach.tex

```diff
@@ -122,9 +126,9 @@ \subsubsection{Optimizations and Edge-Cases}
 In VTK a user can place multiple planes at desired angles to clip the volume. This technique is essential for many medical use-cases. Since only one side of clip planes needs to be traversed, performing any sort of ray cast on the clipped side is wasteful. Hence a simple optimization is to move the starting point of the ray on the plane by projecting the ray onto the plane in the view direction. 
 
 \item Eye Too Close to the Volume
-When the eye is too close to the volume, the near plane could clip the volume visible bounds resulting in OpenGL clipping removing the geometry from rendering. In such cases the bounding box of the volume that defines the texture coordinates and hence define a surface to start casting the ray needs to be clipped. To handle such scenario, a plant-box interaction is added. So far evey move of the camera is funding invokved checking if the geomtry of the volume needs to be clipped.
+When the eye is too close to the volume, the near plane could clip the volume visible bounds resulting in OpenGL clipping removing the geometry from rendering. In such cases the bounding box of the volume that defines the texture coordinates and hence define a surface to start casting the ray needs to be clipped. To handle such scenario, a plane-box intersection is added. So far evey move of the camera is funding invokved checking if the geomtry of the volume needs to be clipped.
```

Comment:
I would actually change the wording here.  Instead of describing it as an edge-case (when the eye is too close to the volume),  I would go for describing the what the intention is for instance instead of,
``` In such cases the bounding box of the volume that defines the texture coordinates and hence define a surface to start casting the ray needs to be clipped```
something like,
```the implementation handles camera views within the volume by clipping the proxy geometry with the near plane (using vtkClipPoly), ensuring all the fragments fall within range."